README.md

```diff
@@ -19,7 +19,13 @@ ADAL for Android gives you the ability to add support for Work Accounts to your
 
 A Work Account is an identity you use to get work done from your organization or school. Anywhere you need to get access to your work life you'll use a Work Account. The Work Account can be tied to an Active Directory server running in your datacenter or live completely in the cloud like when you use Office 365. A Work Account will be how your users know that they are accessing their important documents and data backed my Microsoft security.
 
-## ADAL for Android 1.16.3 Released!
+## ADAL for Android 2.0.0 Released!
+>Please note: ADAL 2.0.0 (released March 2020) is neither API-compatible nor cache-compatible the following versions:<br/>
```

Comment:
add `with` after cache-compatible

Reply:
Good catch, will do

Reply:
Corrected in 12696342ce21a103ab129ade131db976b5ccc2e0